frontend/coprs_frontend/tests/test_logic/test_copr_dirs_logic.py

```diff
@@ -116,4 +116,4 @@ def test_coprdir_build_normal_then_pr(self):
         response = self.test_client.get(url)
         assert response.status_code == 200
         result_dict = json.loads(response.data)
-        assert result_dict["git_repo"] == repo_url.format("test-pr-dirs:pr:1")
+        assert result_dict["git_repo"] == repo_url.format("test-pr-dirs:pr:test-copr-copr-1")
```

Comment:
This puzzles me.  :sweat_smile:  Are we only affecting the src.fp.org-triggered builds, or are we also affecting the normal `foo:pr:1` builds?  People still can submit `copr build test:pr:1 <src.rpm>`, right?

Reply:
I see, this is `PullRequestTrigger` that uses get_pagure_pr_events().  So this test-case is pagure-only.